src/views/templates/details/tabs/scheduling/components/Descheduler.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { TemplateSchedulingGridProps } from 'src/views/templates/details/tabs/scheduling/components/TemplateSchedulingLeftGrid';
+import { useDeschedulerInstalled, useDeschedulerOn } from 'src/views/templates/utils';
+import { DESCHEDULER_URL } from 'src/views/templates/utils/constants';
+
+import { useModal } from '@kubevirt-utils/components/ModalProvider/ModalProvider';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import {
+  Button,
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTermHelpText,
+  DescriptionListTermHelpTextButton,
+  Popover,
+} from '@patternfly/react-core';
+import { ExternalLinkAltIcon, PencilAltIcon } from '@patternfly/react-icons';
+
+import DeschedulerModal from './DeschedulerModal';
+
+import 'src/views/templates/details/tabs/scheduling/TemplateSchedulingTab.scss';
+
+const Descheduler: React.FC<TemplateSchedulingGridProps> = ({ template }) => {
+  const { t } = useKubevirtTranslation();
+  const { createModal } = useModal();
+  const isDeschedulerOn = useDeschedulerOn(template);
+  const isDeschedulerInstalled = useDeschedulerInstalled();
+
+  return (
+    <DescriptionListGroup>
+      <DescriptionListTermHelpText>
+        <Popover
+          hasAutoWidth
+          maxWidth="30rem"
+          headerContent={t('Descheduler')}
+          bodyContent={
+            <>
+              {t(
+                'The descheduler can be used to evict a running VM so that the VM can be rescheduled onto a more suitable node via a live migration.',
+              )}
+              <div className="margin-top">
+                <Button
+                  variant="link"
+                  icon={<ExternalLinkAltIcon />}
+                  href={DESCHEDULER_URL}
+                  target="_blank"
+                  component="a"
+                  iconPosition="right"
+                  className="no-left-padding"
+                >
+                  {t('Learn more')}
+                </Button>
+              </div>
+            </>
+          }
+        >
+          <DescriptionListTermHelpTextButton>{t('Descheduler')}</DescriptionListTermHelpTextButton>
+        </Popover>
+      </DescriptionListTermHelpText>
+
+      <DescriptionListDescription>
+        {useDeschedulerInstalled && (
```

Comment:
you can use here `isDeschedulerInstalled` instead :)

Reply:
I cannot name it `isDeschedulerInstalled` because our rules force me to name it with "use"... 

Also it is by design to disable the option when the Descheduler is not installed.. yes, we are missing some toolip that it is not installed. It is not perfect

Reply:
I meant using the const you decalred on line 26

Reply:
since you are using the hook instead it's return value it will always be true

Reply:
Yes, I can see that the `{useDeschedulerInstalled && ...` check is unnecessary. Thank you very much, Aviv!